src/sage/groups/galois_group.py

```diff
@@ -39,26 +39,17 @@ def _alg_key(self, algorithm=None, recompute=False):
         algorithm = self._get_algorithm(algorithm)
         return algorithm
 
-class _GaloisMixin:
+class _GMixin:
     r"""
     This class provides some methods for Galois groups to be used for both permutation groups
-    and abelian groups.
+    and abelian groups, subgroups and full Galois groups.
+
+    It is just intended to provide common functionality between various different Galois group classes.
     """
-    def _repr_(self):
-        """
-        String representation of this Galois group
+    # This class uses the following attributes, which should be defined in any subclass
+    # * _default_algorithm -- a string, the default algorithm used to compute the Galois group
```

Comment:
Would it not be more straightforward to define a method that throws not implemented instead?

Or pass these in the `__init__`?